AbstractDataService.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC, abstractmethod
+
+from abc import ABC, abstractmethod
+
+class AbstractDataService(ABC):
+
+    @abstractmethod
+    def return_data(self):
+        pass
```

Comment:
could be a bit more descriptive w.r.t. naming. What data is returned? (i.e. weather forecast) You can also add type hints on the expected return type